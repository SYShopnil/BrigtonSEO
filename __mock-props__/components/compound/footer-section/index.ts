import { IFooterSection } from '@src/types/compound';
import { IconEnum } from '@src/types/root';

export const footerSectionData: IFooterSection = {
  headingInfo: {
    title: 'Want something extra?',
    subTitle: 'Subscribe to newsletter',
    buttonText: 'Subscribe',
    buttonLink: '/subscribe',
    linkOpenNewTab: false,
  },
  contactInfo: {
    address:
      'c/o SiteVisibility, Plus X Innovation Hub, Lewes Road Brighton, BN2 4GL - 01273 805525',
    phoneNumberOne: '+965 6554 6598',
    phoneNumberTwo: '+965 6554 6599',
    email: 'brightonseo@gmail.com',
  },
  columnOne: {
    title: 'Company',
    linkFields: [
      {
        label: 'About Us',
        url: '/about-us',
      },
      {
        label: 'Tickets',
        url: '/tickets',
      },
      {
        label: 'Speakers',
        url: '/speakers',
      },
      {
        label: 'Training',
        url: '/training',
      },
      {
        label: 'Sponsors',
        url: '/sponsors',
      },
      {
        label: 'Career',
        url: '/career',
      },
      {
        label: 'Contact',
        url: '/contact',
      },
    ],
  },
  columnTwo: {
    title: 'Support',
    linkFields: [
      {
        label: 'FAQs',
        url: '/faqs',
      },
      {
        label: 'Code of Conduct',
        url: '/code-of-conduct',
      },
      {
        label: 'Privacy Policy',
        url: '/privacy-policy',
      },
      {
        label: 'Terms & Conditions',
        url: '/terms-and-conditions',
      },
      {
        label: 'Unsubscribe',
        url: '/unsubscribe',
      },
    ],
  },
  socialColumn: {
    title: 'Follow Us',
    socialLinks: [
      {
        platform: IconEnum.Facebook,
        url: 'https://www.facebook.com',
      },
      {
        platform: IconEnum.Twitter,
        url: 'https://www.twitter.com',
      },
      {
        platform: IconEnum.Instagram,
        url: 'https://www.instagram.com',
      },
      {
        platform: IconEnum.Youtube,
        url: 'https://www.youtube.com',
      },
    ],
  },
};
