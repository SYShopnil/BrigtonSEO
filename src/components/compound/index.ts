import { ContactSection } from './contact-section';
import { ListContentSection } from './list-content-section';
import BuyTicketCard from './card/buy-ticket';
import { HeroGeneral, HeroHome } from './hero-section';
import CourseCard from './card/course';
import PricingCard from './card/pricing';
import { Testimonial } from './card/testimonial';
import Person from './card/person';
import UpcomingCard from './card/upcoming';
import FooterSection from './footer-section';
import { TestimonialSection } from './testimonial-section';
import { EventHistoryCard } from './card/event-history';
import { TextAccordion } from './textAccordion';
import { ImageAccordion } from './image-accordion';

export {
  ContactSection,
  CourseCard,
  FooterSection,
  ListContentSection,
  Testimonial,
  HeroGeneral,
  BuyTicketCard,
  PricingCard,
  Person,
  EventHistoryCard,
  UpcomingCard,
  TestimonialSection,
  HeroHome,
  TextAccordion,
  ImageAccordion,
};
