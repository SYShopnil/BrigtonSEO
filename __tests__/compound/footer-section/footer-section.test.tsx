/**
 * External Imports
 */
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

/**
 * Internal Imports
 */
import { FooterSection } from '@src/components/compound';
import { footerSectionData } from '@root/__mock-props__/components/compound/footer-section';

/**
 * Test if the footer section is rendering correctly
 */
it('renders footer section correctly', () => {
  const { getByTestId } = render(<FooterSection {...footerSectionData} />);
  const footer = getByTestId('footer-section');
  expect(footer).toBeInTheDocument();
  expect(footer).toMatchSnapshot();
});
