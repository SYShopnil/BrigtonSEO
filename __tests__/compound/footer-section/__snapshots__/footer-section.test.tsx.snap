// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer section correctly 1`] = `
<section
  class="section-padding-primary bg-primaryGradient"
  data-testid="footer-section"
>
  <div
    class="container"
  >
    <div>
      <div
        class="flex flex-col relative"
      >
        <div
          class="flex flex-col md:flex-row md:justify-between items-start md:items-center mb-[3.75rem] md:mb-[7.5rem] "
        >
          <div
            class="z-20"
          >
            <div
              class="section-heading white  "
              data-testid="section-heading"
            >
              <span
                class="sub-title block text-[1.125rem] font-medium leading-[1.5] md:text-[1.5rem]  md:leading-[1.2] capitalize"
              >
                Subscribe to newsletter
              </span>
              <h2
                class="section-title text-[1.75rem] font-medium leading-[1.3] md:text-[3rem] md:font-bold md:leading-[1.1] capitalize"
              >
                Want something extra?
              </h2>
            </div>
          </div>
          <div
            class="mt-[1.5rem] md:mt-0 ml-0 md:ml-auto"
          >
            <a
              class="inline-flex"
              href="/subscribe"
              target="_self"
            >
              <span
                class="bs-button WhiteBgTextBlue hover-GhostBgTextBlue btn-sm   group relative cursor-pointer overflow-hidden inline-flex gap-[1rem] items-center justify-center capitalize"
                data-testid="footer-subscribe-button"
              >
                <span>
                  Subscribe
                </span>
                <span
                  class="bs-btn-icon"
                >
                  <svg
                    fill="none"
                    height="1em"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <line
                      x1="22"
                      x2="11"
                      y1="2"
                      y2="13"
                    />
                    <polygon
                      points="22 2 15 22 11 13 2 9 22 2"
                    />
                  </svg>
                </span>
                <span
                  class="absolute bs-button-bg inset-0"
                />
              </span>
            </a>
          </div>
          <div
            class="absolute xl:scale-[1.825] md:scale-[1.5] right-0 md:right-[2.8125rem] xl:right-[4.625rem] opacity-[10%] md:opacity-[4%] z-0 fill-white"
          >
            <svg
              fill="inherit"
              height="180"
              viewBox="0 0 345 345"
              width="180"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                clip-path="url(#clip0_2197_10104)"
                opacity="1"
              >
                <path
                  d="M345 12.1291C344.101 18.6697 343.356 25.2283 342.287 31.742C334.731 77.8049 319.781 120.921 290.15 157.748C282.1 167.747 272.316 176.363 263.323 185.599C262.55 186.399 261.661 187.082 261.05 187.63C263.233 193.236 265.865 198.492 267.348 204.053C271.633 220.153 268.112 234.78 257.51 247.547C244.851 262.793 228.086 271.616 209.327 276.701C202.031 278.678 194.332 279.226 186.794 280.232C183.614 280.654 181.026 279.288 179.705 276.243C178.394 273.215 179.283 270.592 181.511 268.256C190.046 259.325 193.721 248.58 193.308 236.343C193.137 231.321 191.708 230.512 187.055 232.165C178.609 235.157 170.146 238.122 161.593 240.799C154.19 243.117 153.651 242.947 146.436 241.392C144.154 244.95 141.98 248.589 139.572 252.075C131.54 263.692 119.762 269.9 106.635 273.395C95.7284 276.297 84.5248 278.147 73.3842 280.106C67.9576 281.058 64.2741 277.725 64.7951 272.568C66.3854 256.89 69.458 241.545 75.7291 226.954C80.8862 214.96 89.6369 206.488 100.876 200.307C101.757 199.822 102.646 199.372 103.931 198.69C100.796 191.7 103.491 185.087 105.593 178.52C107.875 171.368 110.544 164.342 113.113 157.29C114.434 153.678 113.508 151.98 109.636 151.755C97.678 151.063 87.0225 154.361 77.9483 162.348C77.526 162.716 77.1217 163.112 76.7174 163.498C74.4174 165.744 71.7491 166.588 68.7303 165.268C65.6756 163.929 64.6334 161.369 64.7143 158.152C64.9928 147.622 67.158 137.461 70.7428 127.587C75.5764 114.245 82.3776 102.063 92.458 91.9463C109.843 74.4986 131.639 71.1654 153.489 82.4947C154.576 83.0607 155.654 83.6447 157.002 84.3635C158.26 82.9439 159.517 81.5514 160.748 80.1318C184.081 53.1428 212.974 34.1139 245.911 20.9877C273.161 10.1256 301.417 3.28848 330.643 0.548242C331.397 0.476367 332.134 0.179883 332.889 -0.00878906C336.932 -0.00878906 340.975 -0.00878906 345.018 -0.00878906V12.1291H345ZM313.995 87.6158C313.537 87.0588 312.98 86.2861 312.315 85.6123C294.696 67.976 277.051 50.3756 259.487 32.6943C258.058 31.2568 256.863 30.9963 255.12 31.8318C243.306 37.492 231.042 42.3705 219.713 48.8752C178.906 72.3064 151.53 107.795 131.342 149.446C130.902 150.345 131.36 152.303 132.106 153.058C151.953 173.066 171.916 192.958 191.789 212.939C193.299 214.457 194.458 214.448 196.156 213.541C202.085 210.36 208.168 207.458 214.008 204.125C260.259 177.702 294.427 140.749 312.98 90.2393C313.249 89.5025 313.546 88.7748 313.995 87.6158ZM79.835 144.478C93.4463 138.413 106.564 136.131 119.797 140.112C128.98 125.153 138.269 110.023 147.577 94.8572C135.475 86.5557 118.369 87.5799 105.899 97.9478C93.2846 108.433 86.1869 122.394 81.5151 137.785C80.9041 139.779 80.4909 141.837 79.835 144.478ZM200.819 264.635C201.34 264.743 201.564 264.86 201.762 264.815C219.614 260.565 235.597 252.946 247.528 238.499C258.265 225.49 257.609 209.237 250.089 197.45C234.932 206.749 219.785 216.047 204.772 225.256C209.3 238.535 206.2 251.59 200.819 264.635ZM79.0354 265.866C92.1885 263.79 104.29 261.185 115.53 255.516C124.487 251.006 130.192 243.54 134.181 234.627C134.613 233.666 134.235 231.725 133.49 230.953C127.102 224.331 120.588 217.826 113.985 211.429C113.257 210.72 111.478 210.315 110.58 210.693C103.653 213.604 97.4354 217.745 93.033 223.873C84.2014 236.172 81.8295 250.709 79.0264 265.857L79.0354 265.866ZM272.9 25.9021C288.326 41.3283 303.735 56.7365 319.188 72.1986C324.902 53.5111 328.954 34.1498 330.885 14.2674C319.574 14.3752 287.473 20.8889 272.9 25.9021ZM179.678 221.061C160.874 202.256 142.591 183.973 123.957 165.34C121.496 173.273 119.034 181.386 116.428 189.454C115.512 192.302 116.339 194.557 118.27 196.506C128.324 206.614 138.359 216.739 148.565 226.685C149.859 227.952 152.384 229.201 153.893 228.805C162.617 226.487 171.206 223.666 179.669 221.07L179.678 221.061Z"
                  fill="inherit"
                />
                <path
                  d="M8.086 345C2.07545 341.136 1.6442 337.093 6.73834 331.99C20.9247 317.786 35.12 303.6 49.3243 289.413C50.1958 288.542 51.0852 287.661 52.1094 286.997C54.7958 285.245 58.2098 285.667 60.42 287.904C62.5942 290.105 62.9985 293.636 61.2286 296.251C60.4829 297.347 59.4946 298.299 58.5423 299.251C44.913 312.899 31.2837 326.546 17.6184 340.157C15.8934 341.873 13.9708 343.392 12.129 345H8.086Z"
                  fill="inherit"
                />
                <path
                  d="M61.9922 345C55.9817 341.136 55.5504 337.093 60.6446 331.99C74.8309 317.786 89.0262 303.6 103.231 289.413C104.102 288.542 104.991 287.661 106.016 286.997C108.702 285.245 112.116 285.667 114.326 287.904C116.5 290.105 116.905 293.636 115.135 296.251C114.389 297.347 113.401 298.299 112.449 299.251C98.8192 312.899 85.1899 326.546 71.5247 340.157C69.7997 341.873 67.877 343.392 66.0352 345H61.9922Z"
                  fill="inherit"
                />
                <path
                  d="M0 278.965C1.53633 277.213 2.98281 275.371 4.61797 273.727C18.3102 259.981 32.0293 246.262 45.7484 232.552C46.6199 231.68 47.4734 230.764 48.4617 230.045C51.0223 228.176 54.5441 228.392 56.8441 230.449C59.2699 232.623 59.8 236.181 58.0031 238.993C57.3473 240.018 56.4578 240.916 55.5953 241.779C41.409 255.983 27.2137 270.178 13.0184 284.365C7.92422 289.45 3.88125 289.018 0.00898437 283.008C0.00898437 281.66 0.00898437 280.313 0.00898437 278.965H0Z"
                  fill="inherit"
                />
                <path
                  d="M163.893 284.068C163.399 284.94 162.752 286.916 161.449 288.264C155.556 294.391 149.5 300.375 143.445 306.349C140.237 309.512 136.032 309.629 133.319 306.799C130.606 303.968 130.75 300.114 133.876 296.952C139.923 290.815 146.032 284.733 152.15 278.659C154.316 276.503 156.858 275.65 159.796 276.934C162.393 278.075 163.758 280.097 163.893 284.068Z"
                  fill="inherit"
                />
                <path
                  d="M36.1802 205.751C36.6294 204.952 37.2583 203.083 38.4981 201.798C44.5087 195.581 50.636 189.481 56.8083 183.425C59.863 180.424 64.0407 180.379 66.7091 183.111C69.3954 185.86 69.3954 189.714 66.4216 192.751C60.2134 199.094 53.9153 205.338 47.5903 211.555C45.488 213.622 42.9364 214.206 40.1513 212.948C37.5458 211.762 36.2521 209.686 36.1802 205.751Z"
                  fill="inherit"
                />
                <path
                  d="M243.054 135.637C224.403 135.691 209.48 120.876 209.363 102.188C209.246 83.5098 224.457 68.2184 243.054 68.3263C261.589 68.4341 276.674 83.5637 276.665 102.027C276.656 120.588 261.652 135.592 243.054 135.637ZM242.911 122.17C254.087 122.197 263.072 113.338 263.179 102.17C263.287 91.0028 254.024 81.7219 242.857 81.8118C231.716 81.9016 222.813 90.895 222.831 102.063C222.849 113.275 231.698 122.143 242.911 122.179V122.17Z"
                  fill="inherit"
                />
                <path
                  d="M194.404 156.921C192.832 155.627 190.343 154.522 189.624 152.716C188.842 150.767 188.842 147.469 190.028 145.96C193.254 141.863 197.126 138.233 201.043 134.756C203.568 132.51 207.207 133.067 209.569 135.412C211.833 137.667 212.543 141.243 210.432 143.642C206.838 147.748 202.849 151.512 198.86 155.259C197.98 156.085 196.443 156.193 194.404 156.921Z"
                  fill="inherit"
                />
                <path
                  d="M176.112 175.375C172.392 175.303 169.589 172.41 169.616 168.664C169.652 164.971 172.581 162.033 176.246 162.024C180.02 162.024 183.092 165.205 182.949 168.96C182.814 172.644 179.831 175.438 176.112 175.366V175.375Z"
                  fill="inherit"
                />
              </g>
              <defs>
                <clippath
                  id="clip0_2197_10104"
                >
                  <rect
                    fill="inherit"
                    height="345"
                    width="345"
                  />
                </clippath>
              </defs>
            </svg>
          </div>
        </div>
        <span
          style="border-image-source: linear-gradient(90deg, rgba(255, 255, 255, 0) 0.76%, #FFFFFF 50.03%, rgba(255, 255, 255, 0) 101.4%); border-image-slice: 1; border-width: 1px;"
        />
      </div>
    </div>
    <div
      class="flex flex-col md:flex-row md:justify-between mt-[3.75rem] md:mt-[7.5rem]"
    >
      <div
        class="max-w-full md:max-w-[25%]"
      >
        <div
          class="flex flex-col"
        >
          <div
            class="fill-white max-w-[75%] md:max-w-full"
          >
            <a
              href="/"
            >
              <svg
                fill="inherit"
                height="100%"
                viewBox="0 0 332 48"
                width="100%"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M15.6907 31.7203C13.7805 31.7203 12.0681 29.1025 12.0681 25.6871C11.8703 21.713 13.6783 19.2518 15.6431 19.2518C17.8533 19.2518 19.2589 22.422 19.2589 25.4351C19.2589 29.2528 17.8058 31.7203 15.6907 31.7203ZM18.761 12.1689C16.2911 12.1689 13.2759 13.2186 12.1707 14.5275L12.1707 6.87508e-05H0L0 6.58545H2.10804L2.10804 31.1684H0L0 37.6989H11.3111L12.0681 35.9882C13.4805 37.1945 14.886 38.2515 18.7064 38.2515C25.8014 38.2515 30.1743 33.0773 30.3243 25.3329C30.4811 18.5974 26.4018 12.1689 18.761 12.1689Z"
                  fill="inherit"
                />
                <path
                  d="M43.7181 15.7817L42.2106 12.564H32.7548V19.1014H34.9174V31.1137H32.7002V37.6987H46.9382V31.1137H44.4207V25.9391C44.4207 21.5151 46.9858 20.3558 51.5157 20.3558V11.3096C46.3309 11.3096 45.028 13.3685 43.7181 15.7817Z"
                  fill="inherit"
                />
                <path
                  d="M60.3706 10.3076C63.9456 10.3076 65.9034 7.99658 65.9034 5.22895C65.9034 2.918 63.843 0.102556 60.3706 0.102556C56.8506 0.102556 54.9404 2.918 54.9404 5.22895C54.9404 7.64226 57.2052 10.3076 60.3706 10.3076Z"
                  fill="inherit"
                />
                <path
                  d="M65.2003 12.564H53.5273V19.1014H55.6423V31.1137H53.5273V37.6987H67.6152V31.1137H65.2003V12.564Z"
                  fill="inherit"
                />
                <path
                  d="M83.7121 31.0658C81.502 31.0658 80.0895 27.8961 80.0895 24.8825C80.0895 21.0649 81.5496 18.5973 83.6645 18.5973C85.5747 18.5973 87.2871 21.2151 87.2871 24.6303C87.4849 28.5983 85.6769 31.0658 83.7121 31.0658ZM87.3347 14.1252C86.3253 13.1231 84.2173 12.0662 81.9997 12.0662C73.704 12.0662 69.3789 17.1928 69.3789 24.9852C69.3789 31.318 72.7967 37.5493 81.4474 37.5493C84.6675 37.5493 86.6322 35.6881 87.3347 34.7815V36.4926C87.3347 38.6059 86.0249 41.2169 81.502 41.2169C77.6746 41.2169 73.9022 39.0078 72.7491 37.9992L70.0819 43.5825C71.9921 45.3411 76.0648 47.9519 81.5976 48C92.165 47.9519 97.2953 42.5258 97.2953 34.1815V19.1497H100.311V12.5641L88.3376 12.5641L87.3347 14.1252Z"
                  fill="inherit"
                />
                <path
                  d="M131.224 22.7216C131.224 17.4452 130.221 11.6642 122.423 11.6642C118.753 11.6642 115.083 14.7795 114.53 15.6862V-0.000130275H102.353V6.5377H104.467V31.1137H102.353V37.6987H118.146V31.1137H114.53V24.0305C114.53 21.8694 115.13 19.354 117.695 19.354C120.663 19.354 120.663 22.3192 120.663 24.0305V37.6987L133.591 37.6987V31.1137H131.224V22.7216Z"
                  fill="inherit"
                />
                <path
                  d="M147.025 4.97642L137.775 7.94208V12.5642H135.06V19.1016H137.775V28.0458C137.925 34.4812 138.177 38.606 145.723 38.606C146.875 38.606 150.3 38.2515 151.808 37.6516V31.2161C150.853 31.6183 149.291 32.068 148.636 32.068C146.677 32.068 147.025 29.7569 147.025 28.0458V19.1016L151.808 19.1016V12.5642L147.025 12.5642V4.97642Z"
                  fill="inherit"
                />
                <path
                  d="M167.159 31.9181C165.051 31.9181 163.441 29.2052 163.441 25.0808C163.441 21.413 164.949 18.5019 166.906 18.5019C169.273 18.5499 170.583 21.4675 170.583 25.2304C170.583 29.0007 169.322 31.9181 167.159 31.9181ZM166.906 11.9644C159.969 11.9644 153.174 16.6411 153.126 25.0327C153.024 33.9295 159.969 38.3535 167.159 38.3535C174.806 38.3535 180.995 33.5747 180.892 25.2304C180.743 16.8386 174.854 12.0664 166.906 11.9644Z"
                  fill="inherit"
                />
                <path
                  d="M210.88 22.7216C210.88 17.4452 209.578 11.6642 201.78 11.6642C198.608 11.6642 196.295 13.3205 194.583 15.6862L193.43 12.564H182.712V19.1014H184.826V31.1137H182.712V37.6987H198.206V31.1137H194.583V23.9759C194.583 21.8149 195.19 19.2995 197.756 19.2995C200.722 19.2995 200.573 22.2646 200.573 23.9759V37.6987L213.248 37.6987V31.1137H210.88V22.7216Z"
                  fill="inherit"
                />
                <path
                  d="M228.51 12.0118C228.51 10.2532 230.523 9.35348 231.983 9.35348C235.504 9.35348 238.416 11.46 238.416 15.182H247.524V2.56339L238.873 2.56339L238.368 6.79022C236.357 3.0202 232.079 1.66374 227.855 1.66374C222.923 1.66374 215.882 5.03102 215.882 12.4688C215.882 25.2849 235.197 23.6283 235.197 27.5486C235.197 29.6551 232.686 30.0573 230.925 30.0573C228.811 29.9618 224.589 29.2528 224.589 25.0327H215.228V37.699H224.035L224.589 33.8268C226.499 36.0905 230.018 38.4561 235.101 38.4561C241.084 38.4561 248.431 35.6406 248.431 27.8008C248.431 15.9866 228.51 17.1929 228.51 12.0118Z"
                  fill="inherit"
                />
                <path
                  d="M250.987 9.20305H254.159V31.0111H250.987V37.6987L281.318 37.6987V26.3413L272.62 26.3413V31.0111L264.571 31.0111V22.824L274.128 22.824V16.1363L264.571 16.1363V9.20305L272.62 9.20305V13.6751H281.318V2.56313L250.987 2.56313V9.20305Z"
                  fill="inherit"
                />
                <path
                  d="M301.305 30.7117C298.084 30.7117 295.723 26.5392 295.723 19.9541C295.723 13.7231 297.832 9.29889 300.902 9.35343C304.777 9.35343 306.435 14.2277 306.435 20.3085C306.435 26.5392 304.476 30.7117 301.305 30.7117ZM300.957 1.60915C290.538 1.60915 283.3 9.10116 283.3 19.9541C283.3 32.525 290.792 38.606 301.305 38.606C312.67 38.606 319.008 31.6657 319.008 20.3085C319.008 8.99876 313.277 1.60915 300.957 1.60915Z"
                  fill="inherit"
                />
                <path
                  d="M320.533 32.4705C320.533 35.1357 322.894 37.897 326.215 37.897C329.989 37.897 332 35.4905 332 32.4705C332 29.9072 329.886 26.7441 326.215 26.7441C322.545 26.7441 320.533 29.9072 320.533 32.4705Z"
                  fill="inherit"
                />
              </svg>
            </a>
          </div>
          <div
            class="mt-[2.5rem]"
          >
            <div>
              <p
                class="text-md md:text-lg font-medium text-white "
              >
                Address:
              </p>
              <div
                class="text-sm md:text-base mt-[0.25rem] text-white"
              >
                <a
                  href="http://maps.google.com/?q=c/o SiteVisibility, Plus X Innovation Hub, Lewes Road Brighton, BN2 4GL - 01273 805525"
                  rel="noreferrer"
                  target="_blank"
                >
                  c/o SiteVisibility, Plus X Innovation Hub, Lewes Road Brighton, BN2 4GL - 01273 805525
                </a>
              </div>
            </div>
          </div>
          <div
            class="mt-[0.5rem] md:mt-[1rem]"
          >
            <div>
              <p
                class="text-md md:text-lg font-medium text-white "
              >
                Phone:
              </p>
              <div
                class="text-sm md:text-base mt-[0.25rem] text-white"
              >
                <a
                  href="tel:+965 6554 6598"
                >
                  +965 6554 6598
                </a>
                 / 
                <a
                  href="tel:+965 6554 6599"
                >
                  +965 6554 6599
                </a>
              </div>
            </div>
          </div>
          <div
            class="mt-[0.5rem] md:mt-[1rem]"
          >
            <div>
              <p
                class="text-md md:text-lg font-medium text-white "
              >
                Email:
              </p>
              <div
                class="text-sm md:text-base mt-[0.25rem] text-white"
              >
                <a
                  href="mailto:brightonseo@gmail.com"
                >
                  brightonseo@gmail.com
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="mt-[2.5rem] md:mt-0"
      >
        <div>
          <div
            class="text-xl font-semibold text-white capitalize"
          >
            Company
          </div>
          <div
            class="flex flex-col mt-[1rem] md:mt-[2.5rem]"
          >
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/about-us"
            >
              About Us
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/tickets"
            >
              Tickets
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/speakers"
            >
              Speakers
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/training"
            >
              Training
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/sponsors"
            >
              Sponsors
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/career"
            >
              Career
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/contact"
            >
              Contact
            </a>
          </div>
        </div>
      </div>
      <div
        class="mt-[2.5rem] md:mt-0"
      >
        <div>
          <div
            class="text-xl font-semibold text-white capitalize"
          >
            Support
          </div>
          <div
            class="flex flex-col mt-[1rem] md:mt-[2.5rem]"
          >
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/faqs"
            >
              FAQs
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/code-of-conduct"
            >
              Code of Conduct
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/privacy-policy"
            >
              Privacy Policy
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/terms-and-conditions"
            >
              Terms & Conditions
            </a>
            <a
              class="text-sm md:text-base leading-[2.2] md:leading-[2.2] text-white self-stretch"
              href="/unsubscribe"
            >
              Unsubscribe
            </a>
          </div>
        </div>
      </div>
      <div
        class="mt-[2.5rem] md:mt-0"
      >
        <div
          class="flex flex-col"
        >
          <div
            class="text-xl font-semibold text-white capitalize"
          >
            Follow Us
          </div>
          <div
            class="flex flex-row gap-x-[1rem] mt-[1.5rem] md:mt-[2.5rem]"
          >
            <a
              class="border p-[0.5rem] border-white text-white rounded-lg scale-125"
              href="https://www.facebook.com"
              target="_blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g>
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M14 13.5h2.5l1-4H14v-2c0-1.03 0-2 2-2h1.5V2.14c-.326-.043-1.557-.14-2.857-.14C11.928 2 10 3.657 10 6.7v2.8H7v4h3V22h4v-8.5z"
                  />
                </g>
              </svg>
            </a>
            <a
              class="border p-[0.5rem] border-white text-white rounded-lg scale-125"
              href="https://www.twitter.com"
              target="_blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g>
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M22.162 5.656a8.384 8.384 0 0 1-2.402.658A4.196 4.196 0 0 0 21.6 4c-.82.488-1.719.83-2.656 1.015a4.182 4.182 0 0 0-7.126 3.814 11.874 11.874 0 0 1-8.62-4.37 4.168 4.168 0 0 0-.566 2.103c0 1.45.738 2.731 1.86 3.481a4.168 4.168 0 0 1-1.894-.523v.052a4.185 4.185 0 0 0 3.355 4.101 4.21 4.21 0 0 1-1.89.072A4.185 4.185 0 0 0 7.97 16.65a8.394 8.394 0 0 1-6.191 1.732 11.83 11.83 0 0 0 6.41 1.88c7.693 0 11.9-6.373 11.9-11.9 0-.18-.005-.362-.013-.54a8.496 8.496 0 0 0 2.087-2.165z"
                  />
                </g>
              </svg>
            </a>
            <a
              class="border p-[0.5rem] border-white text-white rounded-lg scale-125"
              href="https://www.instagram.com"
              target="_blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g>
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M12 2c2.717 0 3.056.01 4.122.06 1.065.05 1.79.217 2.428.465.66.254 1.216.598 1.772 1.153a4.908 4.908 0 0 1 1.153 1.772c.247.637.415 1.363.465 2.428.047 1.066.06 1.405.06 4.122 0 2.717-.01 3.056-.06 4.122-.05 1.065-.218 1.79-.465 2.428a4.883 4.883 0 0 1-1.153 1.772 4.915 4.915 0 0 1-1.772 1.153c-.637.247-1.363.415-2.428.465-1.066.047-1.405.06-4.122.06-2.717 0-3.056-.01-4.122-.06-1.065-.05-1.79-.218-2.428-.465a4.89 4.89 0 0 1-1.772-1.153 4.904 4.904 0 0 1-1.153-1.772c-.248-.637-.415-1.363-.465-2.428C2.013 15.056 2 14.717 2 12c0-2.717.01-3.056.06-4.122.05-1.066.217-1.79.465-2.428a4.88 4.88 0 0 1 1.153-1.772A4.897 4.897 0 0 1 5.45 2.525c.638-.248 1.362-.415 2.428-.465C8.944 2.013 9.283 2 12 2zm0 5a5 5 0 1 0 0 10 5 5 0 0 0 0-10zm6.5-.25a1.25 1.25 0 0 0-2.5 0 1.25 1.25 0 0 0 2.5 0zM12 9a3 3 0 1 1 0 6 3 3 0 0 1 0-6z"
                    fill-rule="nonzero"
                  />
                </g>
              </svg>
            </a>
            <a
              class="border p-[0.5rem] border-white text-white rounded-lg scale-125"
              href="https://www.youtube.com"
              target="_blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g>
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M21.543 6.498C22 8.28 22 12 22 12s0 3.72-.457 5.502c-.254.985-.997 1.76-1.938 2.022C17.896 20 12 20 12 20s-5.893 0-7.605-.476c-.945-.266-1.687-1.04-1.938-2.022C2 15.72 2 12 2 12s0-3.72.457-5.502c.254-.985.997-1.76 1.938-2.022C6.107 4 12 4 12 4s5.896 0 7.605.476c.945.266 1.687 1.04 1.938 2.022zM10 15.5l6-3.5-6-3.5v7z"
                  />
                </g>
              </svg>
            </a>
          </div>
          <div
            class="text-base text-white mt-[3.75rem] md:mt-[5rem]"
          >
            © 2022 brightonSEO.
          </div>
          <div
            class="mt-[1rem]"
          >
            <div
              class="text-base text-white"
            >
              Designed & Developed
              <div>
                by 
                <a
                  class="text-md md:text-base md:font-bold md:italic underline"
                  href="https://www.lemonhive.com/"
                  rel="noreferrer"
                  target="_blank"
                >
                  Lemon Hive.
                </a>
              </div>
            </div>
          </div>
          <div
            class="mt-[1rem]"
          >
            <div
              class="text-base text-white"
            >
              Dev Partner 
              <div>
                <a
                  class="text-md md:text-base md:font-bold md:italic underline"
                  href="https://www.sanity.io/"
                  rel="noreferrer"
                  target="_blank"
                >
                  Sanity.
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
